<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapMenuWidget</class>
 <widget class="QWidget" name="MapMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>121</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar
{
	color: white;
	border-radius: 3;
    background: transparent;
}

QMenuBar::item
{
	spacing: 10px;  /*spacing between menu bar items */
   	padding: 2px 8px;
	border: 1px solid black;
	border-radius: 3;  
	margin-left: 3px;
	/*margin-right: 2px;*/
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
		stop: 0 #6D6D6D, stop: 0.499 #636363,
		stop: 0.5 #595959, stop: 1.0 #595959);
}

QMenuBar::item:selected 
{  /*when selected using mouse or keyboard */
	border-color: white;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
		stop: 0 #BEBEBE, stop: 0.499 #A5A5A5,
		stop: 0.5 #AAAAAA, stop: 1.0 #AAAAAA);
}

QMenuBar::item:pressed 
{
    border: 1px solid white;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
		stop: 0 #3B3B3B, stop: 0.46 #2F2E2E,
		stop: 0.488 #272727, stop: 1.0 #2B2B2B);
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="hideButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>21</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Скрыть меню карты</string>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="ScaleWidget" name="scaleWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>260</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>260</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaleWidget</class>
   <extends>QWidget</extends>
   <header>ScaleWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
